packages/css/src/components/radio/radio.scss

```
@@ -44,111 +23,181 @@
   gap: var(--ams-radio-gap);
   line-height: var(--ams-radio-line-height);
   outline-offset: var(--ams-radio-outline-offset);
+  position: relative;
+  text-decoration-thickness: var(--ams-radio-text-decoration-thickness);
+  text-underline-offset: var(--ams-radio-text-underline-offset);
```

```suggestion
  text-underline-offset: var(--ams-radio-text-underline-offset);
  -webkit-tap-highlight-color: transparent;
```

Think we've already discussed this once but on an Ipad you get an ugly tap effect (semi-transparent black background) and this get rid of that.

---

This is a reset, so let’s include it in the mixin.

Should we check other components for this? Might be applicable with every `touch-action: manipulation;`.

---

Do we want to remove this styling though? It's an a11y feature.

See also [this](https://github.com/necolas/normalize.css/issues/23) and [this](https://github.com/twbs/bootstrap/pull/22964)